manim/mobject/svg/brace.py

```diff
@@ -129,3 +160,40 @@ def change_brace_label(self, obj, *text):
 
 class BraceText(BraceLabel):
     CONFIG = {"label_constructor": Tex}
+
+
+class BraceBetweeenPoints(Brace):
+    """Similar to Brace, but instead of taking a mobject it uses 2
+    points to place the brace. A fitting direction for the brace is
+    computed, but it still can be manually overridden.
+    If the points go from left to right, the brace is drawn from below.
+    Swapping the points places the brace on the opposite side.
```

Comment:
There should be a short-ish one-line description, followed by a newline, and then more detailed description.

```suggestion
    """Similar to Brace, but instead of taking a mobject it uses 2
    points to place the brace. 
    
    A fitting direction for the brace is
    computed, but it still can be manually overridden.
    If the points go from left to right, the brace is drawn from below.
    Swapping the points places the brace on the opposite side.
```